planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/interface.cpp

```diff
@@ -259,6 +259,24 @@ bool LaneChangeInterface::canTransitFailureState()
   }
 
   if (state == LaneChangeStates::Abort) {
+    if (!module_type_->isAbortState() || true) {
```

Comment:
@Kotakku This condition is always true. It doesn't need this cond, I guess.
```suggestion
    {
```

Reply:
I'm sorry. I forgot to erase the true I added for debugging. I need to leave “!module_type_->isAbortState()” to publish only once at the time of abort.

Reply:
resolved https://github.com/autowarefoundation/autoware_universe/pull/10941/commits/b008145288327cf16a76fa60a72022882cce5a5d